### DISCUSSION TITLE
Preprocessor consts

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#122] [General]
Is there an instrinsic preprocessor const which allows conditional compilation of TB specific code?

Can we get a list of all built-in preprocessor consts currently implemented?

I was attempting this
```
' /* _
*/ Option Strict
```
to hack a block comment to ignore the comment line continuation in TB but this apparently does not work now.

**Comment by WaynePhillipsEA** [comment]
The currently defined constants are the same as in VBA:    Mac, Win16, Win32, Win64, VBA6, VBA7

I agree, we should introduce a tB specific one.  'TWINBASIC' would be my vote.

**Comment by Kr00l** [comment]
I also like the idea of having a compilation constant for tB.

Here my ideas (like as brainstorming)

'TB' , it's short but can be misleading maybe (?)
'TWNB' , kind of mix, TWN for TWIN and B for BASIC.
'TWINBASIC', repeat here as I also like it as this cannot be misleading. It's only kinda long, but who cares ?

**Comment by WaynePhillipsEA** [comment]
TWINBASIC it is... in v0.9.2617.  👍

**Reply by wqweto** [reply]
Is it `TWINBASIC = v0.9.2617`?

**Reply by WaynePhillipsEA** [reply]
No, just a flag, so the value is 1 like all the other internal constants.
